### PR TITLE
Crowdloan page UI

### PR DESCRIPTION
**Pull request by AtelyPham**
- Sets up crowdloan page and input
- Fix comments/logs
- Ups
- Fixes
- Validation on amount, below cap and greater than 0.1
- style: crowdloan page
- feat: add statistics and overview section
- style: add background color for statistic card
- style: add active style on sidebar

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] Tests for the changes have been added (for bug fixes/features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## What is the current behavior?

CleanShot 2022-06-14 at 22 31 25

Issue Number: https://github.com/webb-tools/webb-dapp/issues/419

## What is the new behavior?
CleanShot 2022-06-14 at 22 26 31

## Does this PR introduce a breaking change?
- [ ] Yes
- [x] No

## Other information
Checked out from PR https://github.com/webb-tools/webb-dapp/pull/421.

Current missing data:
- [ ] Amout of Contribution in DOT.
- [ ] Total contributors.

### CODE REVIEW

**Comment by dutterbutter** [comment]
@AtelyPham A couple requested changes please. Let's remove the data of raised, cap, current block etc. from the top of the UI card. So specifically this part:

Screen Shot 2022-06-14 at 10 47 57 AM

Let also remove 'Summary' part below the card. I don't think this is necessary at this time, and once we have the docs up regarding our crowdloan we can point the user there for more info.

File: package.json [hunk 1]
```diff
@@ -11,11 +11,13 @@
   },
   "resolutions": {
     "eslint": "^6.6.0",
-    "@polkadot/api": "8.1.1",
-    "@polkadot/util": "9.0.1",
-    "@polkadot/keyring": "9.0.1",
-    "@polkadot/util-crypto": "9.0.1",
-    "@polkadot/wasm-crypto": "6.1.1",
+    "@polkadot/api": "^8.7.1",
```
Comment (nepoche) [inline package.json:14]: Let's hold off on updating these `@polkadot/` packages. They should be synced with the `@webb-tools/` packages.

**Comment by dutterbutter** [comment]
@AtelyPham can you also update the gif / video in the PR description to accurately reflect your changes here please. Great work !!

**Comment by AtelyPham** [comment]
Yeh, of course. Demo for the current behaviour:

https://user-images.githubusercontent.com/60747384/173738574-f8a170b4-3777-45df-b444-a161a11c458f.mp4

**Pull request merged**